src/DynamoCore/Extensions/ExtensionDefinition.cs

```diff
@@ -10,7 +10,7 @@ namespace Dynamo.Extensions
     public class ExtensionDefinition
     {
         [DataMember]
-        public string AssemblyName { get; set; }
+        public string AssemblyPath { get; set; }
```

Comment:
Did you check that the `ExtensionDefinition` xml file for `DynamoPackages` is not broken by this change?